packages/plugins/Pins/src/store/index.ts

```diff
@@ -28,6 +29,21 @@ const getInitialState = (): PinsState => ({
 
 let pinsLayer: VectorLayer<Vector<Geometry>>
 
+function movementNotAllowed({
+  dragging,
+  map,
+  originalEvent,
+}: MapBrowserEvent<PointerEvent>) {
+  if (!dragging && pinsLayer) {
+    pinsLayer
+      .getFeatures(map.getEventPixel(originalEvent))
+      .then(
+        (features) =>
+          (document.body.style.cursor = features.length ? 'not-allowed' : '')
+      )
+  }
```

Comment:
This also changes the cursor when not directly hovering the pin, i.e. hovering its hole or lower left svg box corner. I understand that we're completely rendering in OL's canvas and therefore can't make use of the usual CSS approaches. However, in the [current implementation](https://static.hamburg.de/kartenclient/prod/), we do have highlighting on the features, and it does disappear in the lower left svg box corner; would you please look up that solution? It may be applicable here, too.

Reply:
Implemented in 5eee84cd